controllers/volumereplication_controller.go

```diff
@@ -304,6 +304,16 @@ func (r *VolumeReplicationReconciler) markVolumeAsPrimary(volumeReplicationObjec
 	resp := tasks.RunAll(promoteVolumeTasks)
 
 	isKnownError := r.hasKnownGRPCError(promoteVolumeTasks, resp)
+
+	if !isKnownError {
```

Comment:
Add this in else part for existing if isknownerror

Reply:
Done